src/components/Home.tsx

```diff
@@ -1,42 +1,125 @@
-import React, { useCallback } from "react";
-import styled from "styled-components";
+import React from "react";
+import styled, { css } from "styled-components";
+import Bg from "./assets/home-page/background.jpg";
+import Alpha from "./assets/home-page/360_alpha.png";
+import Oxymore from "./assets/home-page/oxymore.png";
+import Number from "./assets/home-page/number-one.png";
+import Manifesto from "./assets/home-page/manifesto.png";
 import {
-  color,
   space,
+  SpaceProps,
+  layout,
+  LayoutProps,
+  background,
+  BackgroundProps,
+  FlexboxProps,
+  flexbox,
+  PositionProps,
+  position,
+  ColorProps,
+  color,
+  TypographyProps,
   typography,
   border,
-  TypographyProps,
-  ColorProps,
-  SpaceProps,
   BorderProps,
 } from "styled-system";
-import { useTranslation } from "react-i18next";
-import i18next from "i18next";
 
-type HeadingProps = ColorProps & SpaceProps & TypographyProps;
+// background
+type BgProps = BackgroundProps & ColorProps & LayoutProps & FlexboxProps;
 
-const Title = styled.h1<HeadingProps>`
-  ${color}
-  ${space}
-  ${typography}
+const Background = styled.div<BgProps>`
+  ${background};
+  ${color};
+  ${layout};
+  ${flexbox};
 `;
 
-const LangButton = styled.button``;
+// logos and button
+type LogoProps = SpaceProps &
+  LayoutProps &
+  FlexboxProps &
+  PositionProps &
+  ColorProps &
+  SpaceProps &
+  TypographyProps &
+  BorderProps;
+
+const Logo = styled.a<LogoProps>`
+  ${layout};
+  ${position};
+  ${flexbox};
+  ${space};
+  ${position};
+`;
+
+const BuyButton = styled.button<LogoProps>`
+  ${space};
+  ${layout};
+  ${color};
+  ${typography};
+  ${flexbox};
+  ${border}
+  ${position};
+`;
 
 const Home = () => {
-  const { t } = useTranslation();
-  const onLangClick = useCallback(
-    (countryId: string) => () => i18next.changeLanguage(countryId),
-    []
-  );
   return (
-    <main>
-      <Title color="primary" bg="error" p={3} fontSize={3}>
-        {t("home.header")}
-      </Title>
-      <LangButton onClick={onLangClick("en")}>EN</LangButton>
-      <LangButton onClick={onLangClick("es")}>ES</LangButton>
-    </main>
+    <Background
+      backgroundSize="cover"
+      backgroundImage="url('./assets/home-page/background.jpg')"
+      backgroundPosition="center"
+      backgroundRepeat="no-repeat"
+      bg="black"
+      height="100vh"
+      display="flex"
+      justifyContent="center"
+    >
+      <Logo width="30%" position="absolute" left={30} top={30}>
+        <img src={Oxymore} />
+      </Logo>
+
+      <Logo
+        width={[100, 200, 300, 400]}
+        position="absolute"
+        top={[200, 180, 160, 120, 200]}
```

Comment:
Since this image is centered, could we have it statically positioned (ie. default, ie. remove `position="absolute"`), and remove the `top=...`
... and add `alignItems="center"` to `<Background>`

Reply:
(basically because it's impossible to get it right with a bunch of `top` values, because screens come in all different heights, whereas the method i suggest here centers it no matter the height)

Reply:
Thanks for this, works a treat.